### PR TITLE
fast fix of links in readme

### PR DESCRIPTION
**Pull request by riichimaru**
im noob in git so i have no idea how to commit into closed pr 😢

### CODE REVIEW

**Comment by Gabe616** [comment]
im gonna need nexx to review this

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@
 > **Note**
 > If you want to revert to the original XPosed module, just press "Reset module location"
 
-[^1]:Signatures changed, so you'll have to delete original module if you've used Xposed module directly.
+[^1]: Signatures changed, so you'll have to delete original module if you've used Xposed module directly.
```
Comment (Gabe616) [inline README.md:28]: what does this mean

Comment (Gabe616) [reply README.md:28]: like what does the [^1]: do

Comment (riichimaru) [reply README.md:28]: guh mistak again!!

Comment (Gabe616) [reply README.md]: ah right github formats it ok

Comment (riichimaru) [reply README.md]: ^1 is footnote/reference

**Pull request merged**